src/scripts/services/dimStoreService.factory.js

```diff
@@ -570,6 +573,9 @@ function StoreService(
 
         return stores;
       })
+      .then(function(stores) {
+        dimDestinyTrackerService.reattachScores(null, stores);
```

Comment:
I'll rename that method to make it a bit more clear - it's pulling score data from the review data cache (to keep the scores from flickering when the stores are refreshed).